src/scmdata/database/__init__.py

```diff
@@ -10,4 +10,6 @@
 climate models and then load just the ``Surface Temperature`` data for all models.
 """
 
-from ._database import ScmDatabase  # noqa: F401
+from ._database import ScmDatabase
+
+__all__ = ["ScmDatabase"]
```

Comment:
Is this needed? Or just auto-generated by an IDE?

Reply:
Needed.

Without it you get the following mypy error: `Module "scmdata.database" does not explicitly export attribute "ScmDatabase"  [attr-defined]`